bin/compile

```diff
@@ -65,3 +65,4 @@ mkdir -p $BUILD_DIR/.profile.d
 
 set-env PATH '$HOME/.swift-bin:$PATH'
 set-env LD_LIBRARY_PATH '$LD_LIBRARY_PATH:$HOME/.swift-lib'
+set-env SWIFT_BACKTRACE 'enable=yes,sanitize=yes,threads=all,images=all,interactive=no,output-to=stderr,symbolicate=full,swift-backtrace=.swift-bin/swift-backtrace-static'
```

Comment:
I think `symbolicate=fast` is better. It usually finishes in only a few seconds (most i've seen is 5).
`full` can take minutes, like 2-3+. Then there is the chance that heruko just kills the node and users end up with no backtrace which will pretty confusing.

cc @gwynne i'd vote for this change in the template repo too.
Perhaps with a comment to mention `full` exists, above the env.
Or even better, just link to docs right there.

Reply:
I switched it to `fast`.

In my limited testing there was no significant difference, but then I only tested it with a bare template with extra fatalErrors, so I'll trust you on this one.